tests/crc32/main.c

```diff
@@ -52,8 +123,13 @@ main(int argc, char * argv[])
 		}
 	}
 
-	if (failures)
-		return (1);
-	else
-		return (0);
+	if (failures || performance_test())
```

Comment:
I would either put the performance testing into a different utility or invoke it with a flag (like `test_sha256` does).  No reason why `make test` should run a performance test here -- it's not like mpool where we can verify that the performance is as expected.